gallery/src/styles/NxList/NxListExamples.tsx

```diff
@@ -7,18 +7,21 @@
 import React from 'react';
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
+import NxListClickableExample from './NxListClickableExample';
+import NxListWithActionsExample from './NxListWithActionsExample';
+import NxListErrorExample from './NxListErrorStateExample';
+
 const NxListSimpleCode = require('!!raw-loader!./NxListDefaultExample.html').default,
-    NxListClickableCode = require('!!raw-loader!./NxListClickableExample.html').default,
+    NxListClickableCode = require('!!raw-loader!./NxListClickableExample.tsx').default,
```

Comment:
js, jsx, ts, and tsx file extensions are not required in import syntax.
```suggestion
    NxListClickableCode = require('!!raw-loader!./NxListClickableExample').default,
```